.github/workflows/build.yml

```diff
@@ -153,6 +134,7 @@ jobs:
     secrets: inherit
     with:
       upstream-details: ${{ needs.metadata.outputs.upstream-details }}
+      validate-amlg-versions: true
```

Comment:
Will this fail the whole pipeline if it doesn't validate? Want to make sure it does and doesn't just give a warning.